index.html

```diff
@@ -0,0 +1,216 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=Edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="https://kit.fontawesome.com/3b9cd09213.js" crossorgin="anonymous"></script>
+    <link rel="preconnect" href="https:// fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=poppins:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css" <link rel="stylesheet"
+        href="https://fonts.googleleapis.com/css2?family=Material+Symbols+Rounded:ops,wght">
+    <title>MyPorfolio</title>
+</head>
+
+<body>
+    <div class="wraper">
+        <header>
+            <p>My Logo</p>
```

Comment:
- [ ] Also, based on the previous reviewer's suggestions, your logo and hamburger are not well-centered and the hamburger button has a black instead of a blue color. I've attached the comparison for you to have a closer look 👍 

## Your view :
![image](https://user-images.githubusercontent.com/50990925/220542756-668425f5-b7ab-445c-ba70-ab673a120827.png)

## Figma:
![image](https://user-images.githubusercontent.com/50990925/220542950-9f45f25c-8c88-405f-8ddf-c9587b55adc2.png)